README.md

```diff
@@ -73,7 +73,7 @@ $message = $renderer->addBodyToMessage(
 );
 ```
 
-When you use `addBodyToMessage()` method, text body will be generated from the HTML body. If needed, you can pass `textView` parameter with the name of the text view.
+If needed, you can pass `textView` parameter with the name of the text view.
```

Comment:
I think it was pretty convenient because it needs to be done anyway as a fallback for the clients that can not read HTML.

Reply:
Currently it implemented in `MessageSettings`: https://github.com/yiisoft/mailer/blob/1fd665342e771883efb480fb26bfb2638a8f0766/src/MessageSettings.php#L202